scripts/deploy/contracts/InvestorVesting.ts

```diff
@@ -49,11 +49,12 @@ export const deploy = async (deployer, setAddresses) => {
     "InvestorVesting",
     constructorArguments(),
     deployer,
-    6
+    4
```

Comment:
You really shld declare this type of variable so someone else can make sense of it. 
const numberOfFacesInAPyramid = 4. I have no idea what it represents